### PR TITLE
Visitor can view a list of articles on landing page

### PR DESCRIPTION
**Pull request by LauraRe**
#### Description
_As a Visitor 
In order to choose an article to read 
I would like to be able to see the articles listed on a page_

Changes proposed in this pull request:
* added articles (title and lead) on landing page
* added article model and controller
#### What I have learned working on this feature: 
* how to truncate content
#### Screenshots:
landing page

### CODE REVIEW

File: app/views/articles/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+  <h1>The Hub News</h1>
+  <div class="articles">
+    <% @articles.each do |article| %>
+      <div class="article_box w-1/3" >
+        <h3><%= article.title %></h3>
+        <p><%= truncate(article.content, :length => 100, :separator => ' ', :ommision => "... (Read More)") %></p>
```
Comment (tochman) [inline app/views/articles/index.html.erb]: Please use new hash syntax

File: app/views/articles/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+      </div>
+    <% end %>
+  </div>
```
Comment (tochman) [inline app/views/articles/index.html.erb]: Please consider HAML

File: config/routes.rb [hunk 3]
```diff
@@ -1,3 +1,4 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  get 'articles/index'
```
Comment (tochman) [inline config/routes.rb]: Please stick to the `resources` format for defining routes.

File: features/step_definitions/basic_steps.rb [hunk 4]
```diff
@@ -0,0 +1,10 @@
+Given("the following articles exist") do |table|
+    table.hashes.each do |article|
+        FactoryBot.create(:article, article)
```
Comment (tochman) [inline features/step_definitions/basic_steps.rb]: Please include FactoryBot methods to your Cucumber setup to simplify syntax in step definitions

File: features/visitor_can_view_articles_list.feature [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+Scenario: View list of articles on the landing page
+        Given the following articles exist
+            | title                                   | content                                                    |
+            | The Hub News is the site of the moment  | Great articles! This site's popularity is raising so quick!|
+            | Spring hasn't arrived in Sweden yet     | Ice can be still spotted on the street, watch out!         |
+        When I visit the "landing" page
+        Then I should see "The Hub News is the site of the moment"
+        And I should see "Great articles! This site's popularity is raising so quick!"
+        And I should see "Spring hasn't arrived in Sweden yet"
+        And I should see "Ice can be still spotted on the street, watch out!"
```
Comment (tochman) [inline features/visitor_can_view_articles_list.feature]: Please be mindful of code formatting

File: spec/models/article_spec.rb [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+  end
+
+  describe 'Validations' do
+    it { is_expected.to validate_presence_of :title }
+    it { is_expected.to validate_presence_of :content } 
+  end
+
+  describe 'Factory' do
+    it 'should have valid Factory' do
+      expect(FactoryBot.create(:article)).to be_valid
```
Comment (tochman) [inline spec/models/article_spec.rb]: Please include FactoryBot methods to your RSpec setup (`rails_helper`) to simplify syntax in specs

File: app/views/articles/index.html.erb (hunk 2 above)
Comment (LauraRe) [reply app/views/articles/index.html.erb]: done

File: config/routes.rb (hunk 3 above)
Comment (LauraRe) [reply config/routes.rb]: will do

File: features/step_definitions/basic_steps.rb (hunk 4 above)
Comment (LauraRe) [reply features/step_definitions/basic_steps.rb]: done

File: spec/models/article_spec.rb (hunk 6 above)
Comment (LauraRe) [reply spec/models/article_spec.rb]: done

File: app/views/articles/index.html.erb (hunk 1 above)
Comment (LauraRe) [reply app/views/articles/index.html.erb]: done

File: features/visitor_can_view_articles_list.feature (hunk 5 above)
Comment (LauraRe) [reply features/visitor_can_view_articles_list.feature]: done

**Pull request merged**